pkg/asset/agent/image/agentpxefiles.go

```diff
@@ -169,3 +184,24 @@ func (a *AgentPXEFiles) copy(filepath string, src io.Reader) error {
 
 	return nil
 }
+
+func (a *AgentPXEFiles) createiPXEScript(cpuArch, ipxeBaseURL, pxeAssetsFullPath string) error {
+	iPXEScriptTemplate := `#!ipxe 
+	initrd --name initrd %s/%s kernel %s/%s  initrd=initrd coreos.live.rootfs_url=%s/%sg random.trust_cpu=on rd.luks.options=discard ignition.firstboot ignition.platform.id=metal console=tty1 console=ttyS1,115200n8 coreos.inst.persistent-kargs="console=tty1 console=ttyS1,115200n8" 
```

Comment:
Still needs a newline before `kernel`
Where do the `console=` args come from? I don't see them in the CoreOS ISO:

```
$ coreos-installer iso kargs show ~/.cache/agent/image_cache/coreos-x86_64.iso
coreos.liveiso=rhcos-414.92.202304131328-0 ignition.firstboot ignition.platform.id=metal
```

Reply:
The `console=` args were referred from https://github.com/openshift/assisted-service/blob/b633a3ab4b9207aeff323cafcd748442e551d39d/docs/user-guide/deploy-on-bare-metal.md?plain=1#L55 but look like not necessary.